src/main/java/net/io/kino/model/loggingaction/EventData.java

```diff
@@ -0,0 +1,140 @@
+package net.io.kino.model.loggingaction;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import net.io.kino.util.C;
+import net.io.kino.util.DateFormatter;
+import org.springframework.lang.NonNull;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@Entity(name = "logs")
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class EventData {
+
+    @Id
+    @Column
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String authUser;
+
+    @Column
+    @Enumerated(EnumType.STRING)
+    private EventType eventType;
+
+    @Column
+    private LocalDateTime eventDate;
+
+    @Column(nullable = true)
+    private Boolean eventResult;
+
+    @MapKeyColumn(name = "name")
+    @Column(name = "data")
+    @ElementCollection(fetch=FetchType.EAGER)
+    @CollectionTable(name = "additional_logs_data")
+    private Map<String, String> additionalLogData;
+
+    public EventData() { }
+
+    public EventData(String authUser, EventType eventType) {
+        this(authUser, eventType, null, null);
+    }
+
+    public EventData(String authUser, EventType eventType, Boolean result) {
+        this(authUser, eventType, result, null);
+    }
+
+    public EventData(String authUser, EventType eventType, Map<String, String> additionalEventData) {
+        this(authUser, eventType, null, additionalEventData);
+    }
+
+    //todo: Missing user identification
```

Comment:
Tutaj jakieś todo jest. Jest to zrobione ostatecznie?

Reply:
Jeżeli przy wywołaniach funkcji wartością parametru authUser będzie login to `todo` jest do usunięcia

Reply:
Niech będzie login, bo Anita nie dodawała id. U nas w aplikacji i tak są odgórnie tworzeni użytkownicy i nie ma zmiany nazwy użytkownika